### PR TITLE
Starter skeleton for Angular Material docs site

### PR DESCRIPTION
**Pull request by jelbourn**
R: @tinayuangao

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Angular Material Docs Site
+
+## Preview
+https://angular-io-v42.firebaseapp.com/
```
Comment (tinayuangao) [inline README.md:4]: put a placeholder for material-io?

File: LICENSE [hunk 2]
```diff
@@ -9,13 +9,12 @@ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
...
+all copies or substantial portions of the Software.
 
 THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
 IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
 FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
 AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
```
Comment (tinayuangao) [inline LICENSE:20]: nit: `the software` is deleted.

File: angular-cli.json [hunk 3]
```diff
@@ -0,0 +1,59 @@
+{
+  "project": {
+    "version": "1.0.0-beta.19-3",
+    "name": "angular-io-v42"
```
Comment (tinayuangao) [inline angular-cli.json:4]: Do we want to use a different name for material?

**Comment by jelbourn** [comment]
@tinayuangao resolved comments

**Pull request merged**